frontend/src/tests/import-export.test.js

```diff
@@ -190,4 +196,265 @@ describe("Import/export library functionality", () => {
         const ddahJson = ddahData.map(prepareMinimal.ddah);
         expect(ddahJson).toMatchSnapshot();
     });
+
+    it("Import Instructors from JSON/CSV/XLSX", () => {
+        // import correct instructors from XLSX
+        const normalizedSpreadsheetInstructors = normalizeImport(
+            {
+                fileType: "spreadsheet",
+                data: parseSpreadsheet("instructors_correct.xlsx"),
+            },
+            instructorSchema
+        );
+        expect(normalizedSpreadsheetInstructors).toMatchSnapshot();
+        // import correct instructors from JSON
+        const normalizedJsonInstructors = normalizeImport(
+            {
+                fileType: "json",
+                data: importObjectJSONs.instructors,
+            },
+            instructorSchema
+        );
+        expect(normalizedJsonInstructors).toMatchSnapshot();
+        // import instructors data missing required key utorid should throw error
+        expect(() =>
+            normalizeImport(
+                {
+                    fileType: "spreadsheet",
+                    data: parseSpreadsheet(
+                        "instructors_missing_required_keys.xlsx"
+                    ),
+                },
+                instructorSchema
+            ).toThrow(Error)
+        );
+        // compute the difference between existing instructors
+        const instructorsDiff = {
+            initialData: initialInstructorData,
+            modifiedData: normalizedSpreadsheetInstructors,
+            expectedDiff: diffImport.instructors(
+                normalizedSpreadsheetInstructors,
+                initialInstructorData
+            ),
+        };
+        expect(instructorsDiff).toMatchSnapshot();
```

Comment:
Why are you snapshotting the whole object? You should only be snapshotting the diff...